src/main/java/net/hypixel/nerdbot/generator/item/InventoryItem.java

```diff
@@ -3,33 +3,26 @@
 import lombok.Getter;
 import lombok.Setter;
 import lombok.ToString;
-import net.hypixel.nerdbot.generator.image.ImageCoordinates;
-
-import java.awt.image.BufferedImage;
 
 @Getter
 @Setter
 @ToString
 public class InventoryItem extends GeneratedObject {
 
-    private int overallSlot;
-    private int slotInRow;
-    private int row;
-    private ImageCoordinates imageCoordinates;
+    private int[] slot;
+    private int[] amount;
+    private String itemName;
+    private String extraContent;
```

Comment:
See first comment about the "extra content" type stuff, we should have specific variables for the different things IMO